ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies.yaml

```
@@ -93,6 +93,11 @@ tests:
   steps:
     test:
     - ref: stackrox-e2e
+- as: gke-sensor-integration-tests
+  optional: true
```

All of nightlies are postsubmit jobs.
```suggestion
  postsubmit: true
```

---

Added that in [fec3275](https://github.com/openshift/release/pull/40290/commits/fec32759fea1fdc1840a8e55a250542dc96f125c)